pkg/minikube/machine/cache_images.go

```diff
@@ -61,6 +62,57 @@ func CacheImagesForBootstrapper(imageRepository string, version string, clusterB
 	return nil
 }
 
+// LoadImage loads the local image into the container runtime
+func LoadImage(profile, img string) error {
```

Comment:
how about we make this a re-usable function that Cache Add also uses same, so we don't have to maintain both Image Load and Cache Add ?

I suggest adding a --all label to "cache add" command so if that is passed it will load image to all clusters if not, it will add it to the active cluster.